test/e2e/play_kube_test.go

```diff
@@ -803,6 +846,21 @@ var _ = Describe("Podman play kube", func() {
 
 	})
 
+	It("podman play kube fail with custom selinux label", func() {
```

Comment:
You need to add.
```
	if !selinux.GetEnabled() {
		Skip("SELinux not enabled")
       }
```

Reply:
sorry hadn't considered cases where selinux is not enabled, have updated the test